### PR TITLE
Remove 'assignee' and add 'assignees' field.

### PR DESCRIPTION
**Pull request by argonlaser**
Signed-off-by: Venkata krishna Sundararajan <vkvenkat94@gmail.com>

### CODE REVIEW

File: bin/usage.txt [hunk 1]
```diff
@@ -9,7 +9,6 @@ Options:
   -ua, --useragent ua              User agent.
        --title title               Issue title.
        --body content              Issue content.
-       --assignee username         Github username of assigned user.
+       --assignees uname1,uname2    Github usernames of assigned users.
```
Comment (kgryte) [inline bin/usage.txt]: Change to `user1,user2` to match README.

File: lib/data.js [hunk 2]
```diff
@@ -13,7 +13,7 @@ var debug = require( 'debug' )( 'github-create-issue:data' );
 *
 * @param {String} title - issue title
 * @param {String} [opts.body] - issue content
-* @param {String} [opts.assignee] - Github username of an assigned user
+* @param {String} [opts.assignees] - Github username of assigned users
```
Comment (kgryte) [inline lib/data.js]: s/username/usernames/

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "github-create-issue",
-  "version": "1.0.1",
+  "version": "2.0.0",
```
Comment (kgryte) [inline package.json]: Don't bump the version. I do this on my end before publishing to ensure everything gets properly tagged, etc.

File: test/test.validate.js [hunk 4]
```diff
@@ -104,16 +104,17 @@ tape( 'if provided an `assignee` option which is not a primitive string, the fun
...
 		function(){}
 	];
 
 	for ( i = 0; i < values.length; i++ ) {
 		err = validate( {}, {
 			'token': 'abcdefg12345678',
-			'assignee': values[i]
+			'assignees': values[i]
 		});
+		console.log(values[i])
```
Comment (kgryte) [inline test/test.validate.js]: Remove the `console.log`.

File: lib/data.js [hunk 5]
```diff
@@ -28,9 +28,9 @@ function data( title, opts ) {
 		out.body = opts.body;
 		debug( 'Issue body: %s', opts.body );
 	}
-	if ( opts.assignee ) {
-		out.assignee = opts.assignee;
-		debug( 'Assigned user: %s', opts.assignee );
+	if ( opts.assignees ) {
+		out.assignees = opts.assignees;
+		debug( 'Assigned user: %s', opts.assignees );
```
Comment (kgryte) [inline lib/data.js]: I would rather be explicit here.
```
debug( 'Assigned users: %s', opts.assignees.join( ',' ) );
```

**Review by kgryte — CHANGES_REQUESTED** [review]

File: bin/usage.txt (hunk 1 above)
Comment (argonlaser) [reply bin/usage.txt]: Done.

File: package.json (hunk 3 above)
Comment (argonlaser) [reply package.json]: Done.

File: test/test.validate.js (hunk 4 above)
Comment (argonlaser) [reply test/test.validate.js]: Done.

File: lib/data.js (hunk 5 above)
Comment (argonlaser) [reply lib/data.js]: Done.

File: lib/data.js (hunk 2 above)
Comment (argonlaser) [reply lib/data.js]: I have changed the param to ```{String[]}``` from ```{String}```. Correct me if its not what you meant.

File: lib/validate.js [hunk 6]
```diff
@@ -19,7 +19,7 @@ var isStringArray = require( 'validate.io-string-primitive-array' );
 * @param {String} options.token - Github access token
 * @param {String} [options.useragent] - user agent string
 * @param {String} [options.body] - issue content
-* @param {String} [options.assignee] - Github username of an assigned user
+* @param {String} [options.assignees] - Github username of assigned users
```
Comment (kgryte) [inline lib/validate.js]: `@param {String[]}`

**Review by kgryte — CHANGES_REQUESTED** [review]
Awesome! Only two more (JSDoc param type and making `username` plural) changes are needed before merging. Thanks!

File: lib/validate.js (hunk 6 above)
Comment (kgryte) [reply lib/validate.js]: Ah, and

> - Github usernames of assigned users

Comment (argonlaser) [reply lib/validate.js]: Oops. I missed that. Thanks for pointing.

**Review by kgryte — APPROVED** [review]

**Pull request merged**